actors/miner/tests/change_beneficiary_test.rs

```diff
@@ -0,0 +1,411 @@
+            BeneficiaryChange::new(first_beneficiary_id, TokenAmount::from(101), 200),
+            None,
+        ),
+    );
+
+    //beneficiary in approval message must equal with proposal
+    let second_beneficiary_id = Address::new_id(1010);
+    expect_abort(
+        ExitCode::USR_ILLEGAL_ARGUMENT,
+        h.change_beneficiary(
+            &mut rt,
+            first_beneficiary_id,
+            BeneficiaryChange::new(second_beneficiary_id, TokenAmount::from(100), 200),
+            None,
+        ),
+    );
+}
+
+#[test]
+fn fails_proposal_beneficiary_with_invalidate_params() {
+    let (mut h, mut rt) = setup();
+    let first_beneficiary_id = Address::new_id(999);
+
+    //not-call unable to proposal beneficiary
+    expect_abort(
+        ExitCode::USR_FORBIDDEN,
+        h.change_beneficiary(
+            &mut rt,
+            first_beneficiary_id,
+            BeneficiaryChange::new(first_beneficiary_id, TokenAmount::from(100), 200),
+            None,
+        ),
+    );
+
+    //quota must bigger than zero while change beneficiary to address(not owner)
+    expect_abort(
+        ExitCode::USR_ILLEGAL_ARGUMENT,
+        h.change_beneficiary(
+            &mut rt,
+            h.owner,
+            BeneficiaryChange::new(first_beneficiary_id, TokenAmount::from(0), 200),
+            None,
+        ),
+    );
+
+    //quota must be zero while change beneficiary to owner address
+    expect_abort(
+        ExitCode::USR_ILLEGAL_ARGUMENT,
+        h.change_beneficiary(
+            &mut rt,
+            h.owner,
+            BeneficiaryChange::new(h.owner, TokenAmount::from(20), 0),
+            None,
+        ),
+    );
+
+    //expiration must be zero while change beneficiary to owner address
+    expect_abort(
+        ExitCode::USR_ILLEGAL_ARGUMENT,
+        h.change_beneficiary(
+            &mut rt,
+            h.owner,
+            BeneficiaryChange::new(h.owner, TokenAmount::from(0), 1),
+            None,
+        ),
+    );
+}
+
+#[test]
+fn successfully_get_beneficiary() {
+    let (mut h, mut rt) = setup();
+    let mut info = h.get_info(&rt);
+    assert_eq!(h.owner, info.beneficiary);
+    assert_eq!(ChainEpoch::from(0), info.beneficiary_term.expiration);
+    assert_eq!(TokenAmount::zero(), info.beneficiary_term.quota);
+    assert_eq!(TokenAmount::zero(), info.beneficiary_term.used_quota);
+
+    let first_beneficiary_id = Address::new_id(999);
+    let beneficiary_term =
+        BeneficiaryTerm::new(TokenAmount::from(100), TokenAmount::zero(), ChainEpoch::from(200));
+    h.propose_approve_initial_beneficiary(&mut rt, first_beneficiary_id, beneficiary_term.clone())
+        .unwrap();
+
+    info = h.get_info(&rt);
+    assert_eq!(first_beneficiary_id, info.beneficiary);
+    assert_eq!(beneficiary_term.expiration, info.beneficiary_term.expiration);
+    assert_eq!(beneficiary_term.quota, info.beneficiary_term.quota);
+    assert_eq!(TokenAmount::zero(), info.beneficiary_term.used_quota);
```

Comment:
Use get_beneficiary instead of inspecting the state here. Maybe do that for many of the other tests above too. With `derive(Eq)` you may be able to rewrite many of these assertions as a single assertion of equality against a struct literal.